jmeter-java-dsl/src/test/java/us/abstracta/jmeter/javadsl/http/DslHttpSamplerTest.java

```diff
@@ -47,13 +39,26 @@ public void shouldSendPostWithContentTypeToServerWhenHttpSamplerWithPost() throw
     testPlan(
         threadGroup(1, 1,
             httpSampler(wiremockUri).post(JSON_BODY, contentType)
-        )
+        ),
+            resultsTreeVisualizer()
```

Comment:
any particular reason to add a resultsTreeVisualizer here? maybe you left it after doing some tests?